peerconnection.go

```diff
@@ -2083,7 +2083,12 @@ func (pc *PeerConnection) Close() error {
 	// https://www.w3.org/TR/webrtc/#dom-rtcpeerconnection-close (step #7)
 	closeErrs = append(closeErrs, pc.dtlsTransport.Stop())
 
-	// https://www.w3.org/TR/webrtc/#dom-rtcpeerconnection-close (step #8, #9, #10)
+	// https://www.w3.org/TR/webrtc/#dom-rtcpeerconnection-close (step #8)
+	if pc.iceGatherer != nil {
+		closeErrs = append(closeErrs, pc.iceGatherer.Close())
+	}
```

Comment:
The `iceTransport` closes it's own gatherer, but the peerconnection also maintains it's own separate iceGatherer. Should close this one too.